snmp_cisco/metadata.csv

```diff
@@ -0,0 +1,179 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+snmp.devices_monitored,gauge,,,,Devices monitored count. Make 'sum by {X}' queries to count all the Devices with the tag X.,0,cisco,
+snmp.sysUpTimeInstance,gauge,,,,The time (in hundredths of a second) since the network management portion of the system was last re-initialized.,0,cisco,
```

Comment:
Not sure if we should or not list common metrics like those. Will lead to duplication for all vendor tiles.